app/src/main/kotlin/net/corda/cli/application/logger/LoggerStream.kt

```diff
@@ -0,0 +1,65 @@
+package net.corda.cli.application.logger
+
+import java.io.OutputStream
+import java.io.PrintStream
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.io.ByteArrayOutputStream
+
+enum class LogLevel {
+    INFO, ERROR
+}
+
+class LoggerStream(
+    private val logger: Logger,
+    private val logLevel: LogLevel
+) : OutputStream() {
+
+    public companion object {
+        fun redirectSystemAndErrorOut() {
+            val sysOut: Logger = LoggerFactory.getLogger("System Out")
```

Comment:
I actually have another task CORE-6297 for updating the readme and docs. I would suggest developers use the named loggers, but at least we catch system out in case they dont. As for what shouldnt be in the console, we would have to set up another named logger. Since console is set to info by default.